xc/device.ml

```diff
@@ -2489,6 +2485,36 @@ module Backend = struct
             ; xen_platform_device
             ] in
 
+        let disks' =
+          let format_of_media (media:Dm_Common.media) file =
+            match media, file with
+            | Dm_Common.Disk, _   -> ["format=raw"]
+            | Dm_Common.Cdrom, "" -> []
+            | Dm_Common.Cdrom, _  -> ["format=raw"]
+          in
+          let lba_of_media (media:Dm_Common.media) =
+            match media with
+            | Dm_Common.Disk  -> "force-lba=on"
+            | Dm_Common.Cdrom -> "force-lba=off"
+          in
+          List.map (fun (index, file, media) ->
+            let file = file ^ match String.startswith "nbd:unix:" file with
```

Comment:
I probably would have written
```
let file = match String.startswith "nbd:unix:" file with
| true -> file ^ ":exportname=qemu_node"
| false -> file
```
I think it is a bit more obvious what `file` will be.

Reply:
Or `if String.startswith "nbd:unix:" file then file ^ ":exportname=qemu_node" else file` :)

Reply:
I removed this bit as @gaborigloi suggested, as this has been removed from the qemu-wrapper